src/client/client.ts

```diff
@@ -89,6 +90,20 @@ export class DDClient<AuthStateArgs = unknown> {
         this.getContext().then(context => {
             this.syncDebugMode(context);
         });
+
+        this.registerEventListeners();
+    }
+
+    private registerEventListeners() {
+        // Since computing the size of an iframe is hard to do correctly,
+        // we listen to `'resize'` events so we can send the actual values over.
+        window.addEventListener('resize', () => {
+            const dimensions: IFrameDimensions = {
+                height: document.documentElement.scrollHeight,
+                width: document.documentElement.scrollWidth
+            };
+            this.framePostClient.send(EventType.RESIZE_IFRAME, dimensions);
+        });
```

Comment:
I wasn't entirely sure where to put this, but decided the `DDClient` would be a good start. If there's a better place for this (or a different way to do this) more than happy to change it.